addons/mail/static/src/components/composer_text_input/composer_text_input.scss

```diff
@@ -19,7 +25,7 @@
     padding: 10px;
     max-height: 200px;
     resize: none;
-    border-radius: 10px 0 0 10px;
+    border-radius: $o-mail-border-radius-lg 0 0 $o-mail-border-radius-lg;
```

Comment:
I would just do `border-radius: $o-mail-border-radius-lg;` here, and then (didn't check for typos on my example):

```
&.o-composer-is-compact {
    border-radius-top-right: 0
    border-radius-bottom-right: 0
}
```

Idea being to do an override for special cases, and not have negative selectors.

Didn't we also want a rule for the extended selector to avoid rounded on the top?

Reply:
The border we want to avoid in extended are those of `o_Composer_coreMain`